src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/controller/EventManagerController.kt

```diff
@@ -68,35 +70,35 @@ class EventManagerController(private val getEventManager: JpaBasedGetEventManage
   ): CompleteDetails =
     getEventManager.findCompleteDetailsByCrnAndEventNumber(crn, eventNumber) ?: throw EntityNotFoundException("Complete details of event manager not found for crn $crn eventNumber $eventNumber")
 
-  @Operation(summary = "Get allocated events created by logged in user")
+  @Operation(summary = "Get allocated event count by logged in user")
   @ApiResponses(
     value = [
       ApiResponse(responseCode = "200", description = "OK"),
     ],
   )
   @PreAuthorize("hasRole('ROLE_WORKLOAD_MEASUREMENT') or hasRole('ROLE_WORKLOAD_READ')")
-  @GetMapping("/allocation/events/me")
-  suspend fun getAllocationsByLoggedInUser(
+  @GetMapping("/allocation/events/me/count")
```

Comment:
Does the count endpoint also need changing to take into teams.

Reply:
I dont believe so, I think it's used elsewhere... happy to change it if you have knowledge that its used on the same screen